docs/user/04-metrics.md

```diff
@@ -480,6 +480,33 @@ If Volume metrics are enabled, the following metrics are collected:
   - `k8s.volume.inodes.free`
   - `k8s.volume.inodes.used`
 
+If Deployment metrics are enabled, the following metrics are collected:
```

Comment:
In the MetricPipeline example above, can you also add the new selectors as disabled?
just to let the user know how they can be configured in the example

Reply:
fixed